src/components/OSCALCatalogGroup.js

```diff
@@ -1,74 +1,118 @@
-import React from "react";
-import { styled } from "@mui/material/styles";
+import ExpandLess from "@mui/icons-material/ExpandLess";
+import ExpandMore from "@mui/icons-material/ExpandMore";
+import Paper from "@mui/material/Paper";
+import Collapse from "@mui/material/Collapse";
 import List from "@mui/material/List";
 import ListItem from "@mui/material/ListItem";
 import ListItemText from "@mui/material/ListItemText";
-import ListItemAvatar from "@mui/material/ListItemAvatar";
-import Avatar from "@mui/material/Avatar";
-import Box from "@mui/material/Box";
-import Collapse from "@mui/material/Collapse";
-import ExpandLess from "@mui/icons-material/ExpandLess";
-import ExpandMore from "@mui/icons-material/ExpandMore";
-import FolderIcon from "@mui/icons-material/Folder";
+import { styled } from "@mui/material/styles";
+import React from "react";
 import OSCALControl from "./OSCALControl";
-import OSCALControlParamLegend from "./OSCALControlParamLegend";
+
+// Groups may not necessarily have an ID (it is not required per the spec);
+// therefore, we need to be able to come up with a semi-constant ID. All
+// groups will have a title. We can (poorly) hash that hopefully that will
+// be good enough.
+export function getGroupKey(group) {
+  if (group.id) {
+    return group.id;
+  }
+  let hash = 7;
+  // eslint-disable-next-line no-restricted-syntax
+  for (const char of group.title) {
+    hash = 31 * hash + char.charCodeAt(0);
+  }
+  return hash;
+}
```

Comment:
Title is very long and this gives a smaller key based on the title. The React [documentation](https://reactjs.org/docs/lists-and-keys.html) recommends to never use index as a key.

Reply:
I will switch it to just use `title`. I do not think it matters too much if key is long after thinking about it more.

Reply:
It seems `groupKey()` was added [here](https://github.com/EasyDynamics/oscal-react-library/commit/2418ffdd340f09f87d0265116aacfd1be5e491f4). I agree it may just be unneeded complication. I assume that our examples may not have always have unique `titles`.